app/components/hiring_staff/vacancies_component.rb

```diff
@@ -10,15 +10,17 @@ def initialize(organisation:, sort:, selected_type:, filters:, filters_form:)
     @selected_type = selected_type&.to_sym || :published
     @vacancy_types = %i[published pending draft expired awaiting_feedback]
 
-    @vacancies = send(@selected_type)
-    @school_options = set_school_options if organisation.is_a?(SchoolGroup)
+    @vacancies = @organisation.all_vacancies
+    @organisation_options = set_organisation_options if @organisation.is_a?(SchoolGroup)
 
-    @vacancies = set_filters(@vacancies, @filters) if organisation.is_a?(SchoolGroup)
+    @vacancies = Vacancy.in_organisation_ids(@filters[:managed_school_ids]) if @filters[:managed_school_ids]&.any?
```

Comment:
lets change this to `:managed_organisation_ids`

Reply:
This is already a huge PR, I would do that in another one as it requires a migration